Clarify summary_crc begins at Footer record if there are no summary records (#224)
foxglove/mcap: docs/specification/README.md

| --- | --- | --- | --- |
| 8 | summary_start | uint64 | Byte offset of the start of file to the first record in the summary section. If there are no records in the summary section this should be 0. |
| 8 | summary_offset_start | uint64 | Byte offset from the start of the first record in the summary offset section. If there are no Summary Offset records this value should be 0. |

> | 4 | summary_crc | uint32 | A CRC32 of all bytes from the start of the Summary section up through the end of the previous field in the footer record. (If there are no records in the summary section, this range begins at the start of the footer record.) A value of 0 indicates the CRC32 is not available. |

Reviewer: ```suggestion
| 4 | summary_crc | uint32 | A CRC32 of all bytes from the start of the Summary section up through and including the end of the previous field (summary_offset_start) in the footer record. A value of 0 indicates the CRC32 is not available. |
```

Reviewer: Small suggestion - take it or leave it. :D
